googlemock/src/gmock.cc

```diff
@@ -120,6 +127,19 @@ static bool ParseGoogleMockStringFlag(const char* str, const char* flag,
   return true;
 }
 
+static bool ParseGoogleMockIntFlag(const char* str, const char* flag,
```

Comment:
Maybe we should just call testing::internal::ParseInt32Flag instead of rewriting the function.

Reply:
deprecated in favor of upstream google3 cr/

Reply:
@gennadiycivil I think this is ready for another look!